### PR TITLE
Bluebutton 289 Fix remaining issues

### PR DESCRIPTION
**Pull request by dtisza1**
* Remove the -i (interactive) option from the docker-run commands in the loadtest run.sh script. Was receiving an error when running under the Ansible shell module vs. running manually on the instance.
* Fix issue with results.log file timestamp.

### CODE REVIEW

File: playbook/loadtest/main.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   vars:
     ansible_ssh_pipelining: no
     results_fn: "{{ lookup('pipe', 'date +%Y%m%d%H%M') }}_loadtest_results.log"
-    src_results_file: "/home/ec2-user/bluebutton-web-deployment/locust/{{ results_fn }}"
```
Comment (whytheplatypus) [inline playbook/loadtest/main.yml:18]: the flexibility of this being a `var` seems good, why the change to hardcoding?

**Review by whytheplatypus — CHANGES_REQUESTED** [review]
Couple questions

File: playbook/loadtest/main.yml (hunk 1 above)
Comment (dtisza1) [reply playbook/loadtest/main.yml:18]: Was having issues with the timestamps being different, since the date was being executed when the variable was being referenced. So was getting different dates where the var was being used. So just setting the timestamp on the file for the S3 copy.

**Review by whytheplatypus — APPROVED** [review]
